src/lib/swapper/swappers/utils/constants.ts

```diff
@@ -3,3 +3,4 @@ export const ALLOWABLE_MARKET_MOVEMENT = '0.01' // 1%
 export const MAX_ALLOWANCE = '100000000000000000000000000'
 
 export const MINIMUM_DONATION_RECEIVE_AMOUNT_USD_FROM_ETH_CHAIN = '500'
+export const MINIMUM_DONATION_Sell_AMOUNT_USD_FROM_ETH_CHAIN = '500'
```

Comment:
```suggestion
export const MINIMUM_DONATION_SELL_AMOUNT_USD_FROM_ETH_CHAIN = '500'
```

Reply:
Indeed! Fixed in https://github.com/shapeshift/web/pull/4902/commits/d7641e10ca68c0fa631a463acb55c5c0953be8c8